applications/nrf5340_audio/src/main.c

```diff
@@ -195,7 +195,7 @@ void main(void)
 
 #if defined(CONFIG_AUDIO_DFU_ENABLE)
 	/* Check DFU BTN before Initialize BLE */
-	dfu_entry_check();
+	dfu_entry_check((void *) ble_core_init);
```

Comment:
Why do we have to pass this parameter at all? Can we move the `dfu_entry_check` to after we have done the ble_core_init in main? @pirun ?
Can it be located at line 209?

Reply:
Because we need to call ble_core_init with a different callback

Reply:
No, since DFU feature is using:
1. Different callback function
2. Set different ADV name